Course/Course/Data/Network/CourseEndpoint.swift

```diff
@@ -37,6 +39,10 @@ enum CourseEndpoint: EndPointType {
             return url
         case .getCourseDates(let courseID):
             return "/api/course_home/v1/dates/\(courseID)"
+        case .getCourseDeadlineInfo(courseID: let courseID):
```

Comment:
`case .getCourseDeadlineInfo(courseID: let courseID):`
can be simplified:
`case .getCourseDeadlineInfo(let courseID):`